docs/jdbc-mysql-catalog.md

```diff
@@ -51,7 +55,7 @@ Please refer to [Manage Metadata Using Gravitino](./manage-metadata-using-gravit
 ### Schema capabilities
 
 - Gravitino schema corresponds to the MySQL database.
-- Supports create schema with comments.
+- Supports create schema, but does not explicitly support setting comments. 
```

Comment:
Should we also make changes to the MySQL table?

Reply:
mysql table support set comment

Reply:
Remove the trailing whitespace.

Reply:
done